src/ui/units/wizard/containers/Wizard/WizardActionPanel/useWizardActionPanel.ts

```diff
@@ -47,6 +48,28 @@ export const useWizardActionPanel = (
 
     const enableEditHistory = Utils.isEnabledFeature(Feature.EnableEditHistory);
 
+    const onClickGoBack = () => {
+        dispatch(goBack({unitId: WIZARD_EDIT_HISTORY_UNIT_ID}));
+    };
+
+    const onClickGoForward = () => {
+        dispatch(goForward({unitId: WIZARD_EDIT_HISTORY_UNIT_ID}));
+    };
+
+    let undoHotkey;
+    let redoHotkey;
+
+    if (isMacintosh()) {
```

Comment:
Would be great to have it as hook too.
Best what i could think of is shared part for meta\ctrl detect and declarative (object like) config for init:

```
useHotkeys([
  {
    key: [SOME_CONST_META_KEY, 'shift', 'z'],
    handler: onClickGoBack,
    options: { scopes: WIZARD_SCOPE_CONST },
  }
])
```